### PR TITLE
doc-api-login

### PR DESCRIPTION
**Pull request by ghost**
## Description
Docs api login, logout user
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-sonle** [comment]
LGTM

File: docs/api-references/user.md [hunk 1]
```diff
@@ -50,3 +50,72 @@ Get user's infomation
...
+        "user": {
+            "id": 1,
+            "name": "admin",
+            "email": "admin@test.com",
+            "role": 1,
+            "created_at": "2018-06-07 03:41:51",
```
Comment (at-thinhnguyen) [inline docs/api-references/user.md]: xoá ngày đi

File: docs/api-references/user.md [hunk 2]
```diff
@@ -50,3 +50,72 @@ Get user's infomation
...
+            "updated_at": "2018-06-07 03:41:51",
+            "deleted_at": null,
+            "user_info": {
+                "id": 1,
+                "user_id": 1,
```
Comment (at-thinhnguyen) [inline docs/api-references/user.md]: xoá `user_id` và `avatar` nếu dc

**Review by ghost — CHANGES_REQUESTED** [review]
Xoá ngày đi gái

**Pull request merged**